### ISSUE TITLE
Minor translation issue in enu (in synology-autorun-1.10.0-rc7)

### ISSUE THREAD

**Issue by 007revad** [schmidhorst/synology-autorun#1]
Hi Horst,

I have an issue where autorun won't start in "DSM 7.1.1-42962 Update 3". The only log entry I can find just says "System failed to start [Autorun]." I've tried it with "run-as": "root" and "run-as": "package".

I've fixed some misspelt words in **/enu/strings** and **/enu/lang.txt** for you: 
[https://gist.github.com/007revad/b4350b988287d645996c8af2494d6cde](https://gist.github.com/007revad/b4350b988287d645996c8af2494d6cde)

Do you now where I can ask questions about getting DSM 7's synodsmnotify working. I've been trying for weeks to get synodsmnotify to work in my script.

**Comment by 007revad** [comment]
I found autorun's log, and the reason why autorun would not start. **/var/log/tmp/autorun.log** was missing.

```
/var/packages/autorun/scripts/postinst: line 5: /var/log/tmp/autorun.log: No such file or directory
< ... > 
/var/packages/autorun/scripts/start-stop-status: line 12: /var/log/tmp/autorun.log: No such file or directory
```
So I created the missing folder and log file and now autorun starts.

```
/var/log/tmp/
/var/log/tmp/autorun.log
```

**Comment by schmidhorst** [comment]
Hello 007revad, 
thanks for your corrections and your debugging help! 
- The spelling corrections I have taken over.
- I have found that the path /var/log/tmp, which I have used for the debug logfile, is only a link to /var/tmp. And that link seems to be not always existing. So I changed now the folder to /var/tmp. Hopefully that fixes the issue.

Your Issue about the synodsmnotify: It was also hard work for me to make it working. I have not fully understood, whats really necessary.
- The files "package\bin\<packageName>-cli" and "conf\resource" seem to be necessary. I have not understood, what they are doing. I think, that generates the riquired "worker". And a "package\ui\index.cgi" seem to be required.
- The dsmappname in the file INFO needs to be identically to .url in ui/config. In my case both set to SYNO.SDS._ThirdParty.App.autorun
I have worked on the package "DskMsg" to figure out what is required for synodsmnotify. It allows to define some messages (including variables) which can be displayed with the shell command synodsmnotify. I think you know that e.g. 7zip can extract the files from an *.spk file. So the attachment may help you:
[DskMsg.zip](https://github.com/schmidhorst/synology-autorun/files/10290347/DskMsg.zip)
